sys/test_utils/result_output/json/result_output_json.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    test_utils_result_output_json Test result output with JSON
+ * @brief       Enable this module to have results output as json
+ * @ingroup     test_utils_result_output
+ * @{
+ *
+ * @file
+ * @brief       Result output in json
+ *
+ *
+ * @author      Kevin Weiss <kevin.weiss@haw-hamburg.de>
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "fmt.h"
+
+#include "test_utils/result_output.h"
+
+void turo_container_open(turo_t *ctx)
+{
+    (void)ctx;
+    print_str("[");
+}
+
+void turo_s32(turo_t *ctx, int32_t val)
+{
+    (void)ctx;
+    print_s32_dec(val);
+    print_str(",");
+}
+
+void turo_u32(turo_t *ctx, uint32_t val)
+{
+    (void)ctx;
+    print_u32_dec(val);
+    print_str(",");
+}
+
+void turo_s64(turo_t *ctx, int64_t val)
+{
+    (void)ctx;
+    print_s64_dec(val);
+    print_str(",");
+}
+
+void turo_u64(turo_t *ctx, uint64_t val)
+{
+    (void)ctx;
+    print_u64_dec(val);
+    print_str(",");
+}
+
+void turo_float(turo_t *ctx, float val)
+{
+    (void)ctx;
+    print_float(val, 8);
+    print_str(",");
+}
+
+void turo_string(turo_t *ctx, const char *str)
+{
+    (void)ctx;
+    print_str("\"");
+    print_str(str);
+    print_str("\",");
+}
+
+void turo_bool(turo_t *ctx, bool val)
```

Comment:
JSON true/false is `true`/`false`: https://www.w3schools.com/js/js_json_datatypes.asp

Reply:
too much python :smile:  thanks!